ass2/index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1,  maximum-scale=1.0"/>
+    <title>Second UI-tak</title>
+    <link rel="shortcut icon" href="img/favicon.png">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <link rel="stylesheet" type="text/css" href="css/icomoon.css">
+    <link rel="stylesheet" type="text/css" href="fonts/open-sans">
+  </head>
+
+  <body>
+    <div class="navbar-container">
+    <nav class="navbar">
+      <a href="#!" class="brand-logo"><img src="img/logo.png"></a>
+      <div class="menu"><i class="icon-menu"></i></div>
+      <div class="search-container">
+        <form>
+        <label class="icon-search"></label>
+          <input type="text" name="search" placeholder="Lookup User">
+        </form>
+      </div>
+      <div class="clear"></div>
+  </nav>
+  </div>
+
+  <section class="sidebar">
+    <ul class="sidebar-options">
+      <li class="sidebar-name">
+        <div><span>J</span></div>
+        <h2>John Snow </h2>
+        <p>Acme corp</p>
+      </li>
+      <li class="sidebar-menu">Acme corp <i class="icon-share right"></i></li>
+      <li class="sidebar-menu"><i class="icon-briefcase"></i>Organization info</li>
+      <li class="sidebar-menu"><i class="icon-price-tags"></i>Tags</li>
+      <li class="sidebar-menu"><i class="icon-circle-down"></i>Aggregation Units</li>
+      <li class="sidebar-menu"><i class="icon-file-text2"></i>Reports</li>
+      <button class="sidebar-logout flat-button"><i class="icon-switch"></i>Logout</button>
+    </ul>
+  </section>
+
+  <section class="container">
+    <section class="landing-page">
+    <h2 class="landing-heading">List of members in <span>Acme Corp </span>
+    <button class="download"><i class="icon-download"></i></button></h2>
+    <button class="add-member button">Add member</button>
+    <button class="flat-button">View pending invitations</button>
+    <div class="dropdown">
+    <h6>Jump to</h6>
+      <span>
+      <select>
+        <option>J</option>
+        <option>c</option>
+      </select>
+      <span>
+    </div>
+    <div class="clear"></div>
+    <ul class="members-list">
+    <li>A</li>
+      <li> 
+        <div class="letter-style"><h3>A</h3></div>
```

Comment:
@priyanka0901 Class name is not good and `h3` tag for the initial letter. It can be span tag.